### PR TITLE
Peggy create support ticket

### PR DESCRIPTION
**Pull request by pkchau-skylight**
Created a more nicely formatted support ticket template in YAML. Also added 3 severity labels based on the [Atlassian severity levels](https://www.atlassian.com/incident-management/kpis/severity-levels)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/backup-support.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (emmastephenson) [inline .github/ISSUE_TEMPLATE/backup-support.md:1]: I think this template already exists as `support-request.md` - can you edit that file vs making this new one? I don't want to confuse users by having too many support templates.

File: .github/ISSUE_TEMPLATE/support.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: Support
```
Comment (emmastephenson) [inline .github/ISSUE_TEMPLATE/support.yml:1]: Actually if we're switching to the `yaml` files you can probably delete both of the Markdown versions!

**Review by emmastephenson — COMMENTED** [review]
Thanks for editing this Peggy!

File: .github/ISSUE_TEMPLATE/support.yml (hunk 2 above)
Comment (pkchau-skylight) [reply .github/ISSUE_TEMPLATE/support.yml:1]: I think there should only be the support.yml file now but let me know if I'm wrong!

Comment (emmastephenson) [reply .github/ISSUE_TEMPLATE/support.yml:1]: I'm not seeing the other support-request.md removed, but that's ok! We can always do it in a followup PR

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks Peggy!

File: .github/ISSUE_TEMPLATE/support.yml (hunk 2 above)
Comment (pkchau-skylight) [reply .github/ISSUE_TEMPLATE/support.yml:1]: Ah weird I thought i did delete the file but i'll try merging this and then will delete in a separate PR as needed!

**Pull request merged**